skyportal/models/schema.py

```diff
@@ -1373,6 +1373,12 @@ class SpectrumPost(_Schema):
         },
     )
 
+    units = fields.String(
+        metadata={
+            'description': "Units of the fluxes/errors in astropy notation (e.g. erg/s/cm/cm/AA).",
```

Comment:
I'd replace the e.g., with all three options.

Reply:
yup